.build/cassandra-build-deps-template.xml

```diff
@@ -127,5 +127,9 @@
       <groupId>com.fasterxml.jackson.dataformat</groupId>
       <artifactId>jackson-dataformat-yaml</artifactId>
     </dependency>
+    <dependency>
```

Comment:
This needs to be added because `org.apache.cassandra.simulator.debug.SelfReconcilingRandom` uses `import org.hsqldb.lib.IntKeyLongValueHashMap;` from this library. We were using this dependency already as it was among dependencies of Hadoop we removed in this patch so we need to cherry-pick this one only. Scope is `test`.